pkg/arvo/app/link-proxy-hook.hoon

```diff
@@ -209,25 +204,25 @@
     =*  vase  q.cage.sign
     ?+  mark  ~|([dap.bowl %unexpected-mark mark] !!)
       %group-initial  [~ state]
-      %group-update   (handle-group-update !<(group-update:group-store vase))
+      %group-update   (handle-group-update !<(update:group-store vase))
     ==
   ==
 ::
 ++  handle-group-update
-  |=  upd=group-update:group-store
+  |=  upd=update:group-store
   ^-  (quip card _state)
   :_  state
-  ?.  ?=(%remove -.upd)  ~
+  ?.  ?=(%remove-members -.upd)  ~
   ::  if someone was removed from a group, find all link resources associated
   ::  with that group, then kick their subscriptions if they're no longer
   ::
   %-  zing
-  %+  turn  (app-paths-from-group:md %link pax.upd)
+  %+  turn  (app-paths-from-group:md %link (group-id:en-path:group-store group-id.upd))
```

Comment:
Maybe we want to update `+app-paths-from-group:md` to take a `group-id` as argument instead?

Reply:
should probably be done when links is changed to use identifiers tbh